maintenance/wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+* Maintenance script to manage WikiFactory variable to https
+* @usage
+* 	# this will migrate wgUploadPath for wiki with ID 119:
+*   /usr/wikia/backend/bin/run_maintenance '--script=wikia/WikiFactoryVariables/migrateWikiFactoryToHttps.php --dry-run --varName wgUploadPath' --id=119
+*/
+
+ini_set( 'display_errors', 'stderr' );
+ini_set( 'error_reporting', E_NOTICE );
+
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+
+use \Wikia\Logger\WikiaLogger;
+
+/**
+ * Class MigrateWikiFactoryToHttps
+ */
+class MigrateWikiFactoryToHttps extends Maintenance {
+
+	protected $dryRun  = false;
+	protected $verbose = false;
+	protected $varName = '';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Migrates Wiki Factory variable to HTTPS";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+		$this->addOption( 'varName', 'Name of the Wiki Factory variable', true, true, 'k' );
```

Comment:
nit: either dashes or lowerCamelCase, `dry-run` and  `varName` next to each other look bad

Reply:
fix: befce2df384